src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -58,24 +58,25 @@ type StateProps = {
   activeGenomeId: string | null;
   activeEnsObjectId: string | null;
   allActiveEnsObjectIds: { [genomeId: string]: string };
+  allChrLocations: { [genomeId: string]: ChrLocation };
```

Comment:
As @msmanoj reminded me in browser-messaging-service PR, we now have `ChrLocations` type.